tests/test_main.py

```diff
@@ -35,11 +35,29 @@ def test_process_with_valid_config_does_not_error(tidyDataFolders):
             "--config",
             "config.yml",
             "--file",
-            "solo_L2_mag-rtn-ll-internal_20240210_V00.cdf",
+            "imap_mag_l1a_norm-mago_20250502_v000.cdf",
         ],
     )
 
     print("\n" + str(result.stdout))
     assert result.exit_code == 0
     # check that file output/result.cdf exists
     assert Path("output/result.cdf").exists()
+
+
+def test_process_hk_ccsds(tidyDataFolders):
+    result = runner.invoke(
+        app,
+        [
+            "process",
+            "--config",
+            "tests/config/hk.yaml",
+            "--file",
+            "MAG_HSK_PW.pkts",
+        ],
+    )
+
+    print("\n" + str(result.stdout))
+    assert result.exit_code == 0
+    # check that file output/result.cdf exists
+    assert Path("output/result.csv").exists()
```

Comment:
Please can you add a test for headers and one that verifys an expected first row/last row/row count have actually been populated correctly